pkg/compiler/codegen.go

```diff
@@ -2243,13 +2243,16 @@ func (c *codegen) writeJumps(b []byte) ([]byte, error) {
 				return nil, err
 			}
 			if op != opcode.PUSHA && math.MinInt8 <= offset && offset <= math.MaxInt8 {
-				offsets = append(offsets, ctx.IP())
+				offsets = append(offsets, offsetChange{
+					pos:    ctx.IP(),
+					remove: op == opcode.JMPL && offset == longInstructionSize,
```

Comment:
Without this we would have as many jumps as there are nested calls. This affects almost all our current interop calls.
The NOP approach seems easier, though. I think I've seen a NOP used for a debug sequence point (refs. https://github.com/neo-project/neo-debugger/issues/170), but it can be handled separately.